src/renderer/components/database-list-item-metadata.jsx

```diff
@@ -33,10 +47,60 @@ export default class DbMetadataList extends Component {
     }
   }
 
+  shouldComponentUpdate(nextProps, nextState) {
+    if ((this.props.columnsByTable !== nextProps.columnsByTable) ||
+    (this.state.tableheights !== nextState.tableheights) ||
+    this.recalc === true ||
+    nextProps.items !== this.props.items ||
+    this.state.collapsed !== nextState.collapsed) {
```

Comment:
Should not change the `this.state` directly. It can drive into problems to react managing the state changes. Use `this.setState` instead of that.